tests/qam-updinstall/update_install.pm

```diff
@@ -390,6 +390,12 @@ sub run {
             }
         }
 
+        if (is_s390x) {
+            # Make sure that openssh-server-config-disallow-rootlogin is not installed
+            # since in s390 we need to ssh to the system to reconnect to the tty after a reboot
+            zypper_call("rm openssh-server-config-disallow-rootlogin", exitcode => [0, 104], timeout => 1500);
```

Comment:
Hello, one minor comment, `timeout => 1500` is a bit too long to remove a single package.

Reply:
Yes, I also though that, but I thought I'd better leave that number which looked to be used as a kind of standard timeout in most calls in that file.  What would you consider to be a good timeout for this case? 60 seconds?

Reply:
IMO, 60s is good enough, and you can also use the default timeout and don't need to add the args for timeout.

`zypper_call("rm openssh-server-config-disallow-rootlogin", exitcode => [0, 104];`

```
sub zypper_call {
    my $command = shift;
    my %args = @_;
    my $allow_exit_codes = $args{exitcode} || [0];
    my $timeout = $args{timeout} || 700;
    my $log = $args{log};
```

Reply:
ok, done